columnphysics/icepack_fsd.F90

```diff
@@ -620,7 +620,6 @@ subroutine fsd_add_new_ice (n, &
                                   wave_sig_ht,               &
                                   wave_spectrum,             &
                                   wavefreq,                  &
-                                  dwavefreq,                 &
```

Comment:
I guess I am not sure why you are removing dwavefreq from the call, but not wavefreq?